### PR TITLE
Update pe.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This pull request adds Planeta TV, Telesur, Hatun TV, Central TV, Galáctica TV and Vision TV Música. Also, the PR edits Panamericana channel with SD resolution. These are the last ones to be added, there are others but the opencaster problem prevents it from working in independent apps.

### CODE REVIEW

File: channels/pe.m3u [hunk 1]
```diff
@@ -93,3 +95,15 @@ https://live.siete.us/uraniotv/uraniotv/index.m3u8
 https://rtmp02.portalexpress.es/primaveratv/primaveratv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/tvnorte7/picture?width=320&height=320" group-title="Local",TV Norte (Chiclayo)
 https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/TVNORTEHD
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/PlanetaTeleInformativo/picture?width=320&height=320" group-title="Local",Planeta TV (Moyobamba)
+https://media2.cdnlayer.biz:8081/8018/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/telesurexpresion/picture?width=320&height=320" group-title="Local",Telesur (Ilo)
+https://qlobbidev.s.llnwi.net/telesur3/hls/ilo.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwnhI0HKESXrVQ8-ktPCeRap6-cP_4upPtEpAlwPa=s88-c-k-c0x00ffffff-no-rj" group-title="Music",Hatun TV
+https://live.siete.us:19360/bestcablehatuntv/bestcablehatuntv.m3u8
```
Comment (themak1985) [inline channels/pe.m3u:103]: Link Not Working - All the rest are fine.

**Review by themak1985 — CHANGES_REQUESTED** [review]

File: channels/pe.m3u (hunk 1 above)
Comment (sguinetti) [reply channels/pe.m3u]: Fixed

File: channels/pe.m3u [hunk 2]
```diff
@@ -93,3 +95,17 @@ https://live.siete.us/uraniotv/uraniotv/index.m3u8
...
+https://live.siete.us/bestcablehatuntv/bestcablehatuntv/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/visiontvmusica/picture?width=320&height=320" group-title="Music",Visión TV Musica (Peru)
+https://5ee0faac3bbae.streamlock.net/visionmusica/visionmusica/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CentralTVperuLima/picture?width=320&height=320" group-title="Local",Central TV (Chosica)
+https://5c3fb01839654.streamlock.net:1963/iptvcentraltv/livecentraltvtv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="" group-title="",TelSatel Cine TV (Arequipa)
+https://stmv.panel.mivideo.pro/cinetv/cinetv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/RCGalacticatv/picture?width=320&height=320" group-title="Music",Galáctica TV (Peru)
+https://envivo.galacticatv.com:3913/live/galacticalive.m3u8
```
Comment (freearhey) [inline channels/pe.m3u:111]: image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

File: channels/pe.m3u (hunk 2 above)
Comment (sguinetti) [reply channels/pe.m3u]: Link changed

File: channels/pe.m3u [hunk 3]
```diff
@@ -51,8 +51,10 @@ https://stmv.panel.mivideo.pro/mileniuntv/mileniuntv/playlist.m3u8
 http://vs8.live.opencaster.com/20100152275/jcpstream/playlist.m3u8
 #EXTINF:-1 tvg-id="OndaDigital.pe" tvg-name="Onda Digital" tvg-country="PE" tvg-language="" tvg-logo="https://i.imgur.com/3R2emx7.jpg" group-title="Music",Onda Digital
 http://184.154.28.210:1935/ondadigital/ondadigital/playlist.m3u8
-#EXTINF:-1 tvg-id="PanamericanaTV.pe" tvg-name="Panamericana TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uQhEDES.png" group-title="",Panamericana TV [Not 24/7]
+#EXTINF:-1 tvg-id="PanamericanaTV.pe" tvg-name="Panamericana TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uQhEDES.png" group-title="",Panamericana TV (HD) [Not 24/7]
 https://cdnhd.iblups.com/hls/ptv2.m3u8
+#EXTINF:-1 tvg-id="PanamericanaTV.pe" tvg-name="Panamericana TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uQhEDES.png" group-title="",Panamericana TV (SD) [Not 24/7]
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/PanamericanaPTV
```
Comment (freearhey) [inline channels/pe.m3u:57]: image

File: channels/pe.m3u [hunk 4]
```diff
@@ -85,3 +87,57 @@ https://live.siete.us/uraniotv/uraniotv/index.m3u8
...
+https://panel.streamingtv-mediacp.online:1936/8108/8108/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/GacetaUcayalinaTelevision/picture?width=320&height=320" group-title="News",Gaceta Ucayalina
+https://tvsource.gacetaucayalina.com/hls/prueba.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/uraniotv/picture?width=320&height=320" group-title="Local",Uranio TV (Yurimaguas)
+https://live.siete.us/uraniotv/uraniotv/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/radioprimaveramoquegua/picture?width=320&height=320" group-title="Local",Primavera 15 Radiotelevisión (Moquegua)
+https://rtmp02.portalexpress.es/primaveratv/primaveratv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/tvnorte7/picture?width=320&height=320" group-title="Local",TV Norte (Chiclayo)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/TVNORTEHD
```
Comment (freearhey) [inline channels/pe.m3u:121]: image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: channels/pe.m3u (hunk 3 above)
Comment (sguinetti) [reply channels/pe.m3u:57]: This issue is the scope of @LaneSh4d0w

File: channels/pe.m3u (hunk 4 above)
Comment (sguinetti) [reply channels/pe.m3u:121]: @LaneSh4d0w Same

File: channels/pe.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/pe.m3u:57]: Ouch. I really need to see why it blocks like that over time.

Comment (sguinetti) [reply channels/pe.m3u:57]: It is now operational again, however, I am not sure whether to tag [Not 24/7].

File: channels/pe.m3u (hunk 4 above)
Comment (sguinetti) [reply channels/pe.m3u:121]: Solved for now

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

File: channels/pe.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/pe.m3u:57]: I miiiiight know why it randomly crashes. Someone is literally reloading my server for one SINGLE Dailymotion channel every 3 seconds.

**Comment by LaneSh4d0w** [comment]
Yeah, apologies for the random blockages on my host. I think it's due to what I've said above (Someone trying to ask for the same channel every 3 seconds, even when it's not necessary to do so.)

**Comment by sguinetti** [comment]
I understand and thank you for your contribution, this web service that converts links from Dailymotion or Twitch to m3u, which makes it much easier to watch channels which need a comfortable provider for low budget (p.e. Kanade and its switch from VPS to Twitch).

Now @themak1985 needs review for merge this PR.

File: channels/pe.m3u [hunk 5]
```diff
@@ -85,3 +87,57 @@ https://live.siete.us/uraniotv/uraniotv/index.m3u8
...
+https://stmv.panel.mivideo.pro/mileniuntv/mileniuntv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/Inkavisionoficial/picture?width=320&height=320" group-title="Local",Inkavisión
+https://7.innovatestream.pe:19360/inkavision/inkavision.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/vnptv.pe/picture?width=320&height=320" group-title="News",Visión Noticias Perú (VNP)
+https://5ee0faac3bbae.streamlock.net/visionnoticias/visionnoticias/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CajamarcaTV/picture?width=320&height=320" group-title="Local",Cajamarca TV
+https://live.siete.us/cajamarcatv/cajamarcatv/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/radiosanborja/picture?width=320&height=320" group-title="Music",Radio San Borja TV
+https://5c3fb01839654.streamlock.net:1963/iptvsanborja/livesanborjatv/playlist.m3u8
```
Comment (themak1985) [inline channels/pe.m3u:101]: Now this one is offline.

File: channels/pe.m3u [hunk 6]
```diff
@@ -85,3 +87,57 @@ https://live.siete.us/uraniotv/uraniotv/index.m3u8
...
+https://tv.oyotunstream.com:1936/mastertv/mastertv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/televisiontarapoto/picture?width=320&height=320" group-title="Local",Televisión Tarapoto
+https://5ee0faac3bbae.streamlock.net/televisiontpp/televisiontpp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://pbs.twimg.com/profile_images/1331243814548287489/RCVGVH6L_400x400.jpg" group-title="Local",Radio Tropical Tarapoto
+https://5ee0faac3bbae.streamlock.net/raditropical/raditropical/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/geniostvmoyobamba/picture?width=320&height=320" group-title="Local",Genios TV (Moyobamba)
+https://media2.cdnlayer.biz:8081/8016/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/geniostvmoyobamba/picture?width=320&height=320" group-title="Local",CR Television (Moyobamba)
+https://media2.cdnlayer.biz:8081/8020/index.m3u8
```
Comment (themak1985) [inline channels/pe.m3u:111]: Now offline.

File: channels/pe.m3u [hunk 7]
```diff
@@ -85,3 +87,57 @@ https://live.siete.us/uraniotv/uraniotv/index.m3u8
...
+https://qlobbidev.s.llnwi.net/telesur3/hls/tacna.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwnhI0HKESXrVQ8-ktPCeRap6-cP_4upPtEpAlwPa=s88-c-k-c0x00ffffff-no-rj" group-title="Music",Hatun TV
+https://live.siete.us/bestcablehatuntv/bestcablehatuntv/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/visiontvmusica/picture?width=320&height=320" group-title="Music",Visión TV Musica (Peru)
+https://5ee0faac3bbae.streamlock.net/visionmusica/visionmusica/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CentralTVperuLima/picture?width=320&height=320" group-title="Local",Central TV (Chosica)
+https://5c3fb01839654.streamlock.net:1963/iptvcentraltv/livecentraltvtv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="" group-title="",TelSatel Cine TV (Arequipa)
+https://stmv.panel.mivideo.pro/cinetv/cinetv/playlist.m3u8
```
Comment (themak1985) [inline channels/pe.m3u:135]: now offline.

**Comment by sguinetti** [comment]
It seems that this Sunday the channels agreed to temporaly shut down their broadcasts perhaps because their audiences are not watching. I will put [Not 24/7] until they resume broadcasting their programs.

File: channels/pe.m3u (hunk 7 above)
Comment (sguinetti) [reply channels/pe.m3u:135]: Tagged Not 24/7

File: channels/pe.m3u (hunk 6 above)
Comment (sguinetti) [reply channels/pe.m3u:111]: Tagged Not 24/7

File: channels/pe.m3u (hunk 5 above)
Comment (sguinetti) [reply channels/pe.m3u:101]: Tagged Not 24/7

**Comment by sguinetti** [comment]
@themak1985 Do you have any comments about these channels from pull request?

File: channels/pe.m3u (hunk 5 above)
Comment (themak1985) [reply channels/pe.m3u:101]: I don't see a tag

**Comment by themak1985** [comment]
> @themak1985 Do you have any comments about these channels from pull request to fix?

I unresolved the comment @sguinetti - you didn't mark Radio San Borja TV as not 24/7

File: channels/pe.m3u (hunk 5 above)
Comment (sguinetti) [reply channels/pe.m3u:101]: This is because there are two duplicate entries, I will delete one of them.

Comment (sguinetti) [reply channels/pe.m3u:99]: Done

**Review by themak1985 — APPROVED** [review]

**Comment by themak1985** [comment]
You had some trailing whitespaces, i removed them. merging now.

**Pull request merged**